File: alg_warunkowy.cpp
/*
 * alg_warunkowy.cpp
 * 
 */


#include <iostream>
using namespace std;

int main(int argc, char **argv)
{
	int x; // deklaracja i inicjacja
	cout << " Podaj wynik testu: " ;
	cin >> x;
	
		if (x < 0 || x > 60 ) {
	cout << " Błędne dane! " ;
	return  0 ;
}
	// instrukcja warunkowa złożona
		if (x < 20 );
		cout << " Grupa podstawowa " ;
		else  if (x <= 40 );
		cout << " Grupa średniozaawansowana " ;
		else  if (x <= 60 );
		cout << " Grupa zaawansowana " ;
	
	return 0;
}

File: alg_warunkowy.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  alg_warunkowy.py
#  


def main(args):
	
	w = int(input("Podaj liczbę punktów ucznia: "))
	if(w < 0 and w > 60):
		print ("Błędne dane!")
		elif (w < 20):
			print("Grupa podstawowa")
		elif (w >= 20 and w <= 40):
			print("Grupa średniozaawansowana")
		else:
			print("Grupa zaawansowana")
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
